### ISSUE TITLE
Events in DST are shown one hour earlier than expected

### ISSUE THREAD

**Issue by mastermoo** [nhnent/tui.calendar#258]
## Version

1.8.0 | Tue Nov 06 2018

## Development Environment

Chrome, MacOS

## Current Behavior

Events that are created now for April 2019 are shown 1 hour before their actual date.
E.g. an event starting `Wed Apr 10 2019 12:00:00 GMT+0200` is shown in calendar at 11 AM.

<img width="598" alt="screen shot 2019-01-17 at 7 11 59 pm" src="https://user-images.githubusercontent.com/7849822/51339102-d94d2a00-1a8b-11e9-8e7e-f0e4fc7be961.png">

## Expected Behavior

The event above should be displayed at 12PM. There seems to be an issue with the transition to DST Summer Time?

**Comment by coolvijaymca** [comment]
is there any ETA for this?

**Comment by wilsonpinto** [comment]
> is there any ETA for this?

Sorry I cannot promise, but I will give a check tomorrow afternoon

**Comment by mastermoo** [comment]
Any progress or is there is there some kind of workaround? Let me know, if I can be of any help...

**Comment by wilsonpinto** [comment]
@mastermoo I could not found a solution that didn't affect others functions like drag or resize. The problem here is that the events are getting (europe/amsterdam)+2:00 timezone at April when they should be getting +1:00.
This seems to be something related with the offset that should adapt to the DST change but may require more changes...

I would say that the workaround could be that you detect the timezone before creating the event. If is inside the DST time and the event was created before the DST you need to decrease 1 hour to the event show in the correct way. Otherwise just create a normal event and show in the same way.

I will try to investigate a bit more on it

**Comment by mastermoo** [comment]
thanks @wilsonpinto I will probably implement this workaround exactly as you described!

you got it, fingers crossed :)

**Comment by mastermoo** [comment]
ok for future reference, this is what I did and it seems to work pretty nicely:

```javascript
// => in Feb. it's -60, in April -120 (for Europe/Berlin)
currentZone = moment().zone()

// in my case, offset is either -60, 0 or +60
offset = currentZone - moment(event.start).zone()

correctedStart = moment(event.start).add(offset, 'minutes')
correctedEnd = moment(event.end).add(offset, 'minutes')
```

Even drag and drop still seems to work. Maybe this code can even help with a possible fix for the issue?

**Comment by dongsik-yoo** [comment]
https://github.com/nhn/tui.calendar/releases/tag/v1.11.1-rc

Guys! Please check this version to fix DST bugs.

**Comment by wilsonpinto** [comment]
@dongsik-yoo I don't think it solves the issue, I will try create a demo of the problem. Thanks

**Comment by dongsik-yoo** [comment]
I know some issue remains. still open.

**Comment by haringsrob** [comment]
Hi, sorry for bumping, but I just ran into this issue as well.

I am trying to solve by manipulating the time, however this does not feel like it is working.

**Comment by coolvijaymca** [comment]
For me also it is displaying an hour later time, so for an example if I book and event for 9:30 to 10:30, it is displaying entry as 10:30 to 11:30.
![error](https://user-images.githubusercontent.com/27564614/67863404-39342f00-fafa-11e9-91e7-8b0cec544c6a.JPG)
